modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarPage.testcase

```diff
@@ -157,9 +157,9 @@ definition {
 
 		CalendarEvent.disableEventBySiteCalendarName(calendarName = "Liferay DXP");
```

Comment:
Hey @thaiscabrals! Not sure if you saw my previous comment, but I think the failure on CI is being caused by this line right here. It cannot find the Liferay DXP site. Can you update to make sure it is flexible enough to find the site even if it is running on a portal instance? As you can see in the other lines of this test, the site is not hard coded. Let me know if this makes sense or not. Thank you